Dockerfile

```diff
@@ -1,5 +1,32 @@
-FROM node:7.8.0-onbuild
-RUN apt-get update && apt-get install -y mediainfo && rm -rf /var/lib/apt/lists/*
-COPY config.docker.js config.js
+FROM node:7.8.0
+
+# Install runtime dependencies.
+RUN apt-get update && apt-get install -y mediainfo
+
+# Remove state generated by apt-get update.
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
You should should merge this command with the previous one if you don't want the state to be still in an intermediary filesystem layer (or use [multi-stage builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/)).

Reply:
Fixed. Didn't know about multi-stage builds!